src/macfloat.c

```diff
@@ -70,6 +70,10 @@ static void PrintMacfloatToBuf(char *buf, size_t bufsize, Double val, int precis
     }
     else {
         snprintf(buf, bufsize, "%.*" PRINTFFORMAT, precision, val);
+        if(!strchr(buf, '.'))
```

Comment:
Please, if possible use `git clang-format` on your pull request. If you need help with that, let me know. Alternatively, just adjust to the "desired" formatting:
```suggestion
        if (!strchr(buf, '.')) {
```

Reply:
Sure, I will stick to that.